### ISSUE TITLE
Inference Prompt  and  Long Generation Results

### ISSUE THREAD

**Issue by Ga-Lee** [NVlabs/LongLive#1]
Thanks for your awesome works and contribution to the community! 
I am trying to reproduce LongLive's results in the long video generation task and met some issues during the inference process.

(1) If my understanding is correct, the prompts used for inference is `prompts/vidprom_filtered_extended.txt`,  which also are used for ODE initialization and Self Forcing training as stated in the paper of [Self Forcing ](https://arxiv.org/pdf/2506.08009) (Section 4);  In other words,  We are training and evaluating model with same text prompts;   Is my understanding correct?

(2) For 30s video generation evaluation, I use some text prompts including `"Animated scene features a close-up of a short fluffy monster kneeling beside a melting red candle. The art style is 3D and realistic, with a focus on lighting and texture. The mood of the painting is one of wonder and curiosity, as the monster gazes at the flame with wide eyes and open mouth. Its pose and expression convey a sense of innocence and playfulness, as if it is exploring the world around it for the first time. The use of warm colors and dramatic lighting further enhances the cozy atmosphere of the image." ` (which is also one of the demos shown on[ project page](https://nvlabs.github.io/LongLive/#long)) . But I got some confusing results.  For instance:

https://github.com/user-attachments/assets/e518ec9d-1e80-499a-b7c3-d67e53844688

It seems some quality issues such as flicking occur when video length gets extended.

I am worried whether I got some settings wrong during inference, could you please help me to check with it? 

longlive_inference.yaml I used:

`denoising_step_list:
- 1000
- 750
- 500
- 250
warp_denoising_step: true # need to remove - 0 in denoising_step_list if warp_denoising_step is true
num_frame_per_block: 3
model_name: Wan2.1-T2V-1.3B
model_kwargs:
  local_attn_size: 12
  timestep_shift: 5.0
  sink_size: 3

data_path: prompt/demo.txt
output_folder: videos/long
inference_iter: -1
num_output_frames: 120
use_ema: false
seed: 0
num_samples: 1
save_with_index: false
global_sink: true
context_noise: 0

generator_ckpt: longlive_models/models/longlive_base.pt

adapter:
  type: "lora"
  rank: 256                    # LoRA rank (typical values: 8, 16, 32, 64)
  alpha: 256                   # LoRA alpha (typically same as rank, but can be different)
  dropout: 0.0                # LoRA dropout rate
  dtype: "bfloat16"          # Data type for LoRA parameters: "bfloat16", "float16", "float32"
  verbose: false              # Whether to print all target module names`

Thank you again for your excellent work !

**Comment by Tzy010822** [comment]
I encountered a similar issue. The generated videos exhibited noticeable flickering, and for some prompts, the last 5-10 seconds of the video had a significant decrease in quality.

https://github.com/user-attachments/assets/6df38e84-b869-4cef-9e24-464a0eec07f6

https://github.com/user-attachments/assets/0d753480-df69-418f-b02c-531a7f2257eb

**Comment by AndysonYs** [comment]
> I encountered a similar issue. The generated videos exhibited noticeable flickering, and for some prompts, the last 5-10 seconds of the video had a significant decrease in quality.
> 
>  rank0-0-0_regular.mp4 
>  rank0-31-0_regular.mp4

Hi! Thanks for your questions. Did you load the lora weight correctly? i see the file name ends with "_regular.mp4", which means lora is not enable. With lora, it shold be "xxx_lora.mp4". Maybe there are some peft version issues?

**Comment by AndysonYs** [comment]
> Thanks for your awesome works and contribution to the community! I am trying to reproduce LongLive's results in the long video generation task and met some issues during the inference process.
> 
> (1) If my understanding is correct, the prompts used for inference is `prompts/vidprom_filtered_extended.txt`, which also are used for ODE initialization and Self Forcing training as stated in the paper of [Self Forcing ](https://arxiv.org/pdf/2506.08009) (Section 4); In other words, We are training and evaluating model with same text prompts; Is my understanding correct?
> 
> (2) For 30s video generation evaluation, I use some text prompts including `"Animated scene features a close-up of a short fluffy monster kneeling beside a melting red candle. The art style is 3D and realistic, with a focus on lighting and texture. The mood of the painting is one of wonder and curiosity, as the monster gazes at the flame with wide eyes and open mouth. Its pose and expression convey a sense of innocence and playfulness, as if it is exploring the world around it for the first time. The use of warm colors and dramatic lighting further enhances the cozy atmosphere of the image." ` (which is also one of the demos shown on[ project page](https://nvlabs.github.io/LongLive/#long)) . But I got some confusing results. For instance:
> 
>  rank0-Animated.scene.features.a.close-up.of.a.short.fluffy.monster.kneeling.beside.a.melting.red.candle.T-0.mp4 
> It seems some quality issues such as flicking occur when video length gets extended.
> 
> I am worried whether I got some settings wrong during inference, could you please help me to check with it?
> 
> longlive_inference.yaml I used:
> 
> `denoising_step_list:
> 
> * 1000
> * 750
> * 500
> * 250
>   warp_denoising_step: true # need to remove - 0 in denoising_step_list if warp_denoising_step is true
>   num_frame_per_block: 3
>   model_name: Wan2.1-T2V-1.3B
>   model_kwargs:
>   local_attn_size: 12
>   timestep_shift: 5.0
>   sink_size: 3
> 
> data_path: prompt/demo.txt output_folder: videos/long inference_iter: -1 num_output_frames: 120 use_ema: false seed: 0 num_samples: 1 save_with_index: false global_sink: true context_noise: 0
> 
> generator_ckpt: longlive_models/models/longlive_base.pt
> 
> adapter: type: "lora" rank: 256 # LoRA rank (typical values: 8, 16, 32, 64) alpha: 256 # LoRA alpha (typically same as rank, but can be different) dropout: 0.0 # LoRA dropout rate dtype: "bfloat16" # Data type for LoRA parameters: "bfloat16", "float16", "float32" verbose: false # Whether to print all target module names`
> 
> Thank you again for your excellent work !

Thanks for the thoughtful question! 

(1):  We’re not evaluating on the training prompts. Although prompts/vidprom_filtered_extended.txt is part of the training corpus (as in Self-Forcing/ODE-init setups), LongLive trains only 3,000 iters × bs 64 = 192,000 samples on a ~200k dataset—i.e., <1 epoch, so each item is seen at most once, which largely mitigates overfitting. For quantitative results we use VBench, whose prompts do not overlap with vidprom, so it’s not evaluate-on-train. For qualitative showcases, the long-video set is mixed: some prompts come from vidprom, but many do not—e.g., “fluffy monster,” “chef in kitchen,” and all 240s cases are not from vidprom, and we don’t observe a clear gap between seen vs. unseen prompts. For the interactive long showcase, prompts #2–#6 are all not from the training set, and for the first prompt most are also not from vidprom; specifically, row1 (1,3), row2 (2,3), row3 (1,3), and all of row4 & row5 are not from vidprom.

(2) I think it is the same issue as @Tzy010822 . I test it again and the generation result is free of flickering. The lora weight may not be loaded correctly. Please pull our code again and check peft version. I've just forced the users to enable lora.

**Comment by AndysonYs** [comment]
You are welcome! Feel free to re-open this issue when needed.
